lib/index.ts

```diff
@@ -134,9 +134,9 @@ export class Adapter extends EventEmitter {
 
     packet.nsp = this.nsp.name;
     const encodedPackets = this.encoder.encode(packet);
-
+    const isString = typeof encodedPackets[0] == "string";
```

Comment:
I'm not a javascript guru, but seeing the [typeof examples](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof#examples) shouldn't it be `const isString = typeof encodedPackets[0] === "string";` ?